scripts/test_lib.sh

```diff
@@ -335,6 +346,96 @@ function go_test {
   fi
 }
 
+# run_go_tests_expanding_packages [arguments to pass to go test]
+# Expands the packages in the list of arguments, i.e. ./... into a list of
+# packages for that given module. Then, it calls run_go_tests with the expanded
+# packages.
+function run_go_tests_expanding_packages {
+  local packages=()
+  local args=()
+  for arg in "$@"; do
+    if [[ "${arg}" =~ ./ ]]; then
+      packages+=("${arg}")
+    else
+      args+=("${arg}")
+    fi
+  done
+
+  # Expanding patterns (like ./...) into list of packages
+  local unpacked_packages=()
+  while IFS='' read -r line; do unpacked_packages+=("$line"); done < <(
+    go list "${packages[@]}"
+  )
+
+  run_go_tests "${unpacked_packages[@]}" "${args[@]}"
+}
```

Comment:
This function isn't used at all. Why add it?

Reply:
It's added for compatibility with the old `go_tests` function. It's going to be used by integration tests.